mqtt/mqtt-broker/src/broker.rs

```diff
@@ -35,7 +36,7 @@ pub struct Broker<N, Z> {
     authorizer: Z,
 
     #[cfg(feature = "__internal_broker_callbacks")]
-    pub on_publish: Option<Sender<std::time::Duration>>,
+    pub on_publish: Option<tokio::sync::mpsc::UnboundedSender<std::time::Duration>>,
```

Comment:
nit: remove namespaces

Reply:
I did it intentionally because the code is hidden behind feature